1Password/onepassword-sdk-go
# Fix missing id validation bug

## Summary
This PR resolves a reported bug of item CRUD where the vault and item ids were not correctly validated and supplying invalid IDs resulted in an unfriendly error.

## How to test

Create an item in code (use the example file) and use a vault name instead of a vault id. You should now see a useful message instead of `panic: error when retrieving vault metadata`.
